src/ensRegistry.ts

```diff
@@ -109,6 +110,9 @@ function _handleNewOwner(event: NewOwnerEvent, isMigrated: boolean): void {
   if (domain.name == null) {
     // Get label and node names
     let label = ens.nameByHash(event.params.label.toHexString());
+    if (label == null) {
+      label = nameByHashLocal(event.params.label.toHexString());
+    }
```

Comment:
Note: all that extra logic does is include the `linea` label in the healing logic, since [the graph's `ens.nameByHash`](https://github.com/graphprotocol/graph-node/blob/master/store/postgres/src/primary.rs#L1779-L1788) doesn't know about the word `linea`